File: Editorial/0068_Text_Justification/CSharp/1_Approach.cs
private List<string> GetWords(int i, string[] words, int maxWidth) {
    var currentLine = new List<string>();
    int currLength = 0;
    while (i < words.Length && currLength + words[i].Length <= maxWidth) {
        currentLine.Add(words[i]);
        currLength += words[i].Length + 1;
        i++;
    }

    return currentLine;
}

File: Editorial/68_Text_Justification/JavaScript/1_Approach.js
function getWords(i, words, maxWidth) {
    let currentLine = [];
    let currLength = 0;
    while (i < words.length && currLength + words[i].length <= maxWidth) {
        currentLine.push(words[i]);
        currLength += words[i].length + 1;
        i++;
    }
    return currentLine;
}
